src/datamodule.py

```diff
@@ -175,7 +184,7 @@ def train_dataloader(self):
             self.trn_ds,
             batch_size=self.batch_size,
             num_workers=self.num_workers,
-            shuffle=True,
+            shuffle=False,
```

Comment:
We should be shuffling the mini-batches in the `train_dataloader`, is there a reason to set it to False here?
```suggestion
            shuffle=True,
```

Reply:
I rolled back changes to the datamodule in this PR, so it is the same as in main. I think this should be examined in a separate PR since no training is occurring for the purpose of this one.